radis/lbl/loader.py

```diff
@@ -1284,7 +1288,8 @@ def _check_database_params(
 
         # Check input types are correct
         if isinstance(path, string_types):  # make it a list
-            path = [path]
+            path = get_files_from_regex(path)
```

Comment:
Actually the get_files_from_regex is not called if there was a list initially @CorentinGrimaldi